src/types/pivot_runtime.ts

```diff
@@ -20,6 +20,7 @@ export interface Pivot<T = PivotRuntimeDefinition> {
   onDefinitionChange(nextDefinition: PivotCoreDefinition): void;
 
   getTableStructure(): SpreadsheetPivotTable;
+  getNonCollapsedTableStructure(): SpreadsheetPivotTable;
```

Comment:
maybe it should be the other way around and `getTableStructure` should return the non-collapsed table, it seems to be a bit more used. Or change `getTableStructure` to `getCollapsedTableStructure` so there's no confusion ?

Reply:
Good idea, updated :)